src/idv_service/session/create/sdk.config.builder.js

```diff
@@ -151,6 +151,37 @@ class SdkConfigBuilder {
     return this;
   }
 
+  /**
+   * Sets the biometric consent flow to "EARLY"
+   *
+   * @returns {this}
+   */
+  withEarlyBiometricConsentFlow() {
+    return this.withBiometricConsentFlow(IDVConstants.EARLY);
+  }
+
+  /**
+   * Sets the biometric consent flow to "JUST_IN_TIME"
+   *
+   * @returns {this}
+   */
+  withJustInTimeBiometricConsentFlow() {
+    return this.withBiometricConsentFlow(IDVConstants.JUST_IN_TIME);
+  }
+
+  /**
+   * Sets the biometric consent flow
+   *
+   * @param {string} biometricConsentFlow the biometric consent flow
+   *
+   * @returns {this}
+   */
+  withBiometricConsentFlow(biometricConsentFlow) {
+    Validation.isString(biometricConsentFlow, 'biometricConsentFlow');
```

Comment:
Do we have any `oneOf` in the `Validation` ?
Feel that we could allow any value right now, and that method is by nature public.

Reply:
If you're implying validation on the value passed to this method by the Relying Business, this should be left to the backend.  It also allows older SDKs to provide new values that the backend may support, even if there isn't a "sugar" method for it.

Reply:
Yup, thanks, as discussed, we shall not try to validate the value itself then.